src/TouristInfoCards.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+function TouristInfoCards() {
```

Comment:
This looks nice @ybayraktutan, well done!

Reply:
Thank you